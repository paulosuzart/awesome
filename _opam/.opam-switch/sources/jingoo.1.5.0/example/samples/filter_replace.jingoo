{{ replace ("ho", "hi", "hoge") }}
{{ replace ("日本", "英", "日本語") }}
{{ replace ('"\\([^"]*\\)"', '--- \\1 ---', '"FOO"') }}
