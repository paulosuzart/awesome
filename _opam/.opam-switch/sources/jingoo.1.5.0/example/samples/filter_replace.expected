hige
英語
--- FOO ---
